aexpect/remote_door.py

```diff
@@ -749,6 +752,7 @@ def recreate_exception(class_name, class_dict):
         exceptiontype = RemoteCustomException
         exception = exceptiontype(*class_dict["args"])
         # in the case of non-custom exceptions the class is properly restored
+        # pylint: disable=W0201
```

Comment:
How about simply adding the `__customclass__` attribute on line 749 with the value None?

Reply:
I went with the least intrusion approach... I did not want to introduce behavior change in a "prepare to release / get CI green" kind of PR.

Reply:
Yes, and as we already have a few pylint comments like this left to resolve in the future, we can just consider this as one of them as well. It is best to test before doing greater changes than this.

Reply:
I could add more unit tests in the future but right now this should be enough for the release.

Reply:
My problem is that I spent a lot of time fixing and polishing aexpect. Saying "left to resolve in the future" without a name and a deadline doesn't seem valid to me. Adding the attribute on class-level should be more explicit.

Reply:
@ldoktor I don't see why "left to resolve in the future" is a problem. Everybody uses the same words here, this is the same way the aexpect release was specified to me - as something that will happen "soon". But in contrast to the fact that it has already taken more than half a year, you have already seen that I execute what I say and in a timely manner. In fact, my greatest problem with aexpect so far has been not about me taking too much time when I commit to a task but to me committing to a task that then blocked me from progressing for another year.

Considering the breakage last minutes changes could create and the fact the code in its current version is already in use, I would prefer the first option.

Reply:
Btw, I already have a PR with some small improvements and some extra unit tests coming with these that I haven't pushed considering that aexpect is already overwhelmed with a few PRs with new features and we were waiting for a release in order to be able to finish the migration on the Avocado VT side first before taking further steps here.

Reply:
> My problem is that I spent a lot of time fixing and polishing aexpect. Saying "left to resolve in the future" without a name and a deadline doesn't seem valid to me. Adding the attribute on class-level should be more explicit.

I understand the effort you put in, but please consider that we're addressing the current state: CI is broken, and we're possibly not catching *other issues*. Also, we're attempting to do a release.

I've created an issue to track the overall pylint status: https://github.com/avocado-framework/aexpect/issues/64